### PR TITLE
fix SwitchCase indent rule in eslint

### PR DESCRIPTION
**Pull request by Nokel81**
Signed-off-by: Sebastian Malton <sebastian@malton.name>

### CODE REVIEW

**Review by panuhorsmalahti — APPROVED** [review]
Could we merge this after https://github.com/lensapp/lens/pull/1450?

**Comment by Nokel81** [comment]
> Could we merge this after #1450?

Yup, will run fix again after you merge

**Pull request merged**
